core/templates/dev/head/services/contextual/UrlService.js

```diff
@@ -17,27 +17,62 @@
  * functions on $window to be mocked in unit tests.
  */
 
-oppia.factory('UrlService', ['$window', function($window) {
-  return {
-    getUrlParams: function() {
-      var params = {};
-      var parts = $window.location.href.replace(
-        /[?&]+([^=&]+)=([^&]*)/gi, function(m, key, value) {
-          params[key] = value;
+oppia.factory('UrlService', [
+  '$window', '$location',
+  function(
+      $window, $location) {
```

Comment:
Move this to end of previous line; they can fit there.

Reply:
Done